mcu_cfg.hjson

```diff
@@ -41,7 +42,9 @@
             offset:  0x00030000,
             length:  0x00010000,
             ch_length:    0x100,
-            num_channels:   0x1,
+            num_channels:   0x4,
```

Comment:
by default must be 1, in the CI it could be 4 or 2

Reply:
No problem! I forgot, how do I change the CI mcu-gen configuration?

Reply:
I guess there is a sed command somewhere (ask @JuanSapriza )